builtin/game/register.lua

```diff
@@ -370,13 +384,29 @@ end
 
 local function make_registration()
 	local t = {}
-	local registerfunc = function(func) table.insert(t, func) end
+	local registerfunc = function(func)
+		table.insert(t, func)
+		core.callback_origins[func] = {
+			mod = core.get_current_modname() or "??",
+			name = debug.getinfo(1, "n").name or "??"
+		}
+		--local origin = core.callback_origins[func]
+		--print(origin.name .. ": " .. origin.mod .. " registering cbk " .. tostring(func))
+	end
 	return t, registerfunc
 end
 
 local function make_registration_reverse()
 	local t = {}
-	local registerfunc = function(func) table.insert(t, 1, func) end
+	local registerfunc = function(func)
+		table.insert(t, 1, func)
+		core.callback_origins[func] = {
```

Comment:
Yeah, but what's the point.  I could just fit this assignment into one or two lines, which would be the same making a separate function.